docs/api/product_variants.rst

```diff
@@ -0,0 +1,738 @@
+Product Variants API
+====================
+
+These endpoints will allow you to easily manage product variants. Base URI is `/api/v1/products/{productId}/variants/`.
```

Comment:
All fields returned in response are snake_cased. Moreover, we want to configure all forms to use snake_case as well, thats my concern.